staking/stakingprecompile.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Staking Precompile
+description: Moonbeam Parachain Staking Ethereum Solidity Precompile Interface Demo
+---
+
+# Staking Precompile
+
+![Staking Moonbeam Banner](/images/staking/staking-stake-banner.png)
+
+## Moonbeam Parachain-Staking
+
+A delegated proof of stake pallet recently debuted called [Parachain-Staking](https://github.com/PureStake/moonbeam/tree/master/pallets/parachain-staking/src), allowing token holders (nominators) to express exactly which collator candidates they would like to support and with what quantity of stake. The design of the Parachain-Staking pallet is such that it enforces shared risk/reward on chain between delegators and collators.
+
+The Staking module is coded in Rust and it is part of a pallet that is normally not accessible from the Ethereum side of Moonbeam. However, a Staking Precompile allows developers to access the staking features as a precompiled contract located at address 0x0000000000000000000000000000000000000800. The Staking Precompile is included in our [Moonbase Alpha v8 release](https://moonbeam.network/announcements/testnet-upgrade-moonbase-alpha-v8/).
+
+## The Parachain-Staking Solidity Interface
+
+[StakingInterface.sol](https://github.com/PureStake/moonbeam/blob/master/runtime/precompiles/src/StakingInterface.sol) is an interface through which solidity contracts can interact with Parachain-Staking. The beauty is that solidity developers don’t have to learn the Substrate API. Instead, they can interact with staking functions using the Ethereum interface they are familiar with.
```

Comment:
```suggestion
[StakingInterface.sol](https://github.com/PureStake/moonbeam/blob/master/precompiles/parachain-staking/StakingInterface.sol) is an interface through which solidity contracts can interact with Parachain-Staking. The beauty is that solidity developers don’t have to learn the Substrate API. Instead, they can interact with staking functions using the Ethereum interface they are familiar with.
```

Reply:
We restructured the precopmiles recently since a lot more will be added. The old link still exists for now but will be removed once all runtimes have been upgraded. This one will stick around.